### DISCUSSION TITLE
SSH changed CTRL-C and up and down keys behaviour recently

### DISCUSSION THREAD

**Discussion by crrodriguez** [git-for-windows/git#3651] [Q&A] [answered]
I use the SSH.exe client shipped with git for windows for regular tasks, not running from bash but from powershell (or cmd, for this case does not matter)

In most recent update a few days ago  "2.35.0.windows.1" ssh to any host and press control-C ends the whole ssh session (disconnects from host) it is no longer interpreted as a order to be sent to the remote system's shell .and up and down keys appear not work correctly through the shell history on the remote side.

This only happen when ssh is run on powershell or cmd, it does not happen when run on git bash... is anyone else seeing this ?

Thanks.

**Comment by dscho** [comment, marked as answer]
Is this the same issue as https://github.com/msys2/msys2-runtime/issues/83?

**Comment by crrodriguez** [comment]
@dscho yes, that's almost certainly the same problem

**Reply by dscho** [reply]
Please test the latest [snapshot](https://wingit.blob.core.windows.net/files/index.html). It almost certainly fixes the problem.

**Reply by crrodriguez** [reply]
> Please test the latest [snapshot](https://wingit.blob.core.windows.net/files/index.html). It almost certainly fixes the problem.

It worked fine, thanks,
